Sources/DataConnectSettings.swift

```diff
@@ -16,6 +16,7 @@ import Foundation
 
 @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
 public struct DataConnectSettings: Hashable, Equatable {
+  static let version = "11.3.0-beta"
```

Comment:
This LGTM! Even in a separate file, I think the find string we use in the find and replace command would look about the same.

Reply:
We should do this in a separate file. DataConnectSettings represents the server settings for DC.

Reply:
Done.